backend/src/backend/tests.py

```diff
@@ -40,3 +40,22 @@ def test_list_tasks_unauthorized_user():
     response = client.get("/api/tasks")
     assert response.status_code == 401
     assert response.json() == {"detail": "User not logged in"}
+
+
+# Chat enpoint
+# Test for malformed request
+def test_chat_endpoint_malformed_request():
+    response = client.post("/api/chat", json={"test": "test"})
+    assert response.status_code == 400
+    assert "type" in response.text
+    assert "loc" in response.text
+    assert "msg" in response.text
```

Comment:
Please add more specific assertions, especially `msg` because it's the most important information clients get.

```
assert response.json() == {
  "detail": [
    {
    }
  ]
}
```